src/Sylius/Component/User/Model/User.php

```diff
@@ -141,26 +146,39 @@ public function getId()
     {
         return $this->id;
     }
-
+    
```

Comment:
Should be reverted and same goes to [this](https://github.com/Sylius/Sylius/pull/5547/files#diff-7de2676163182603a1b59ec9f1291164R157).